ui/plugin-runtime/package.json

```diff
@@ -28,7 +28,8 @@
     "lint:fix": "eslint --fix src --ext .ts,.tsx"
   },
   "dependencies": {
-    "@module-federation/enhanced": "^0.1.21"
+    "@module-federation/enhanced": "^0.1.21",
+    "@perses-dev/plugin-system": "0.49.0"
```

Comment:
do you foresee in a long term that `plugin-system` and `plugin-runtime` are merged and there is only one package for the plugin system ?

Reply:
yes I started as its own package to avoid breaking the current system, prior an initial release we can merge them